Sources/Rendering/OpenGL/Glyph3DMapper/index.js

```diff
@@ -694,8 +694,14 @@ function vtkOpenGLGlyph3DMapper(publicAPI, model) {
     const garray = model.renderable.getMatrixArray();
 
     const pts = model.renderable.getInputData(0).getPoints();
-    const { useShiftAndScale, coordShift, coordScale } =
-      computeCoordShiftAndScale(pts);
+    let useShiftAndScale = false;
+    let coordShift = 0.0;
+    let coordScale = 1.0;
+
+    if (model.coordShiftAndScaleEnabled && pts) {
```

Comment:
why don't you check the value of the original mapper (vtkPolyDataMapper.coordShiftAndScaleEnabled), but just the value of the API specific value (vtkOpenGLPolyDataMapper.coordShiftAndScaleEnabled)

Reply:
I see what you mean. Good catch 👍